test/chromedriver-specs.js

```diff
@@ -109,6 +109,51 @@ describe('chromedriver', function () {
         binPath.should.eql('/path/to/chromedriver-36');
       });
 
+      it('should correctly determine Chromedriver versions', async function () {
+        sandbox.stub(fs, 'glob')
+          .returns([
+            '/path/to/chromedriver-36',
+            '/path/to/chromedriver-35',
+            '/path/to/chromedriver-34',
+            '/path/to/chromedriver-33',
+            '/path/to/chromedriver-32',
+            '/path/to/chromedriver-31',
+            '/path/to/chromedriver-30',
+          ]);
+        sandbox.stub(tp, 'exec')
+          .onCall(0)
+            .returns({
+              stdout: 'ChromeDriver 2.36.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(1)
+            .returns({
+              stdout: 'ChromeDriver 2.35.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(2)
+            .returns({
+              stdout: 'ChromeDriver 2.34.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(3)
+            .returns({
+              stdout: 'ChromeDriver 2.33.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(4)
+            .returns({
+              stdout: 'ChromeDriver 2.32.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(5)
+            .returns({
+              stdout: 'ChromeDriver 2.31.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            })
+          .onCall(6)
+            .returns({
+              stdout: 'ChromeDriver 2.30.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8)',
+            });
+
+        const chromedrivers = await cd.getChromedrivers({});
+        _.map(chromedrivers, cd => cd.version).should.eql(['2.36', '2.35', '2.34', '2.33', '2.32', '2.31', '2.30']);
```

Comment:
I'd rather use simple `for of` loop

Reply:
Do you mean something like what I have below?

```
const expectedChromedriverVersions = ['2.36', '2.35', '2.34', '2.33', '2.32', '2.31', '2.30'];
for (let i in chromedrivers) {
    chromedrivers[i].should.eql(expectedChromedriverVersions[i]);
}
```

Reply:
I mean 

```js
for (const [cdVersion, expectedVersion] of _.zip(chromedrivers, [....])) {
  cdVersion.should.eql(expectedVersion);
}
```

Reply:
Thanks for clarifying. I updated accordingly